src/throttle/amoc_throttle_controller.erl

```diff
@@ -260,16 +260,17 @@ continue_plan(Name, State, Info, #change_rate_plan{rates = [Rate]} = Plan) ->
     TRef = Plan#change_rate_plan.timer,
     Info1 = do_change_rate(Name, Rate, Interval, Info),
     {ok, cancel} = timer:cancel(TRef),
-    consume_all_timer_ticks({change_plan, Name}),
+    consume_all_messages({change_plan, Name}),
```

Comment:
revert this change, "consume_all_timer_ticks" is more readable name for this function